src/pip/_internal/commands/freeze.py

```diff
@@ -27,46 +28,54 @@ def __init__(self, *args, **kw):
         super(FreezeCommand, self).__init__(*args, **kw)
 
         self.cmd_opts.add_option(
-            '-r', '--requirement',
+            '-r',
+            '--requirement',
             dest='requirements',
             action='append',
             default=[],
             metavar='file',
             help="Use the order in the given requirements file and its "
-                 "comments when generating output. This option can be "
-                 "used multiple times.")
+            "comments when generating output. This option can be "
+            "used multiple times.",
+        )
         self.cmd_opts.add_option(
-            '-f', '--find-links',
+            '-f',
+            '--find-links',
             dest='find_links',
             action='append',
             default=[],
             metavar='URL',
-            help='URL for finding packages, which will be added to the '
-                 'output.')
+            help='URL for finding packages, which will be added to the ' 'output.',
+        )
         self.cmd_opts.add_option(
-            '-l', '--local',
+            '-l',
+            '--local',
             dest='local',
             action='store_true',
             default=False,
             help='If in a virtualenv that has global access, do not output '
-                 'globally-installed packages.')
+            'globally-installed packages.',
+        )
         self.cmd_opts.add_option(
             '--user',
             dest='user',
             action='store_true',
             default=False,
-            help='Only output packages installed in user-site.')
+            help='Only output packages installed in user-site.',
+        )
         self.cmd_opts.add_option(
             '--all',
             dest='freeze_all',
             action='store_true',
             help='Do not skip these packages in the output:'
-                 ' %s' % ', '.join(DEV_PKGS))
+            ' %s' % ', '.join(DEV_PKGS),
+        )
```

Comment:
Why is the continuation of the help string not indented under the help keyword? This seems like a bug in black.

Reply:
Indeed. @ambv is this the same as https://github.com/ambv/black/issues/147?

Reply:
Yea, I think that's just that issue. Adding our own parenthesis here would make this get formatted more like you'd expect, but once https://github.com/ambv/black/issues/147 gets solved, this should (I believe) format how you expect even without the parenthesis.